<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings - FumoEdit-QT</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="BbMain">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="LblWrapBody">
       <property name="text">
        <string>Word wrap:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="CbWrapPreview">
         <property name="text">
          <string>Body field</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CbWrapBody">
         <property name="text">
          <string>Body preview</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LblBodyFontSize">
       <property name="text">
        <string>Body field font size:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
       <property name="buddy">
        <cstring>SbBodyFontSize</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="SbBodyFontSize">
       <property name="suffix">
        <string>pt</string>
       </property>
       <property name="minimum">
        <number>7</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="LblPreviewFontSize">
       <property name="text">
        <string>Body preview font size:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
       <property name="buddy">
        <cstring>SbPreviewFontSize</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="SbPreviewFontSize">
       <property name="suffix">
        <string>pt</string>
       </property>
       <property name="minimum">
        <number>7</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="LblSitePath">
       <property name="text">
        <string>Path to FumoNet repository:</string>
       </property>
       <property name="buddy">
        <cstring>LeSitePath</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="FilenameEdit" name="LeSitePath"/>
       </item>
       <item>
        <widget class="QPushButton" name="PbBrowseSitePath">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="LblSitePathDescription">
       <property name="text">
        <string>This is used to preview post thumbnails and verify the existence of the filenames that are typed into certain fields.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilenameEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets.FilenameEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>CbWrapPreview</tabstop>
  <tabstop>CbWrapBody</tabstop>
  <tabstop>SbBodyFontSize</tabstop>
  <tabstop>SbPreviewFontSize</tabstop>
  <tabstop>LeSitePath</tabstop>
  <tabstop>PbBrowseSitePath</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>BbMain</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BbMain</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
